pontoon-to-webext.js

```diff
@@ -0,0 +1,164 @@
+#! /usr/bin/env node
+
+/* eslint-disable promise/avoid-new */
+
+const propertiesParser = require('properties-parser');
+const path = require('path');
+const FS = require('q-io/fs');
+const argv = require('minimist')(process.argv.slice(2));
+
+const Habitat = require('habitat');
```

Comment:
Where / how is this getting used? I don't see anything other than `Habitat.load()` around here